server/api.js

```diff
@@ -115,6 +117,37 @@ router.post("/graduates", function (req, res) {
     ],
     (error, result) => {
       if (result) {
+        Connection.query(
+          `select id from graduates where github_id=$1`,
+          [github_id],
+          (error, result) => {
```

Comment:
errors are not handled anywhere, if you have an error you have no chance to get know what happened, since all errors are suppressed you even aren't aware of something went wrong